src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -259,14 +239,6 @@ private class TaskLauncherActor(
           offerOperationAcceptTimeout -= instanceId
         }
         logger.info(s"Synced single $instanceId from InstanceTracker: $instance")
-
-        // Request delay for new run spec config.
-        if (!launchAllowedAt.contains(instance.runSpec.configRef)) {
-          // signal no interest in new offers until we get the back off delay.
-          // this makes sure that we see unused offers again that we rejected for the old configuration.
-          unregister()
-          rateLimiterActor ! RateLimiterActor.GetDelay(instance.runSpec.configRef)
```

Comment:
So we don't `unregister()` anymore?

Reply:
We do every time `manageOfferMatcherStatus` is called.

Reply:
I thought a little more about it and we have to decide what behaviour we want. Currently, we back-off if we do not have any delay update. I changed the code so that we accumulate all updates for a run spec not just the scheduled versions. So we always should have the latest back-off data. Maybe we should use the `Source[(Snapshot, Source[Update])]` pattern again.